--- cryptopp-5.6.5/GNUmakefile.orig	2017-10-17 02:56:28.000000000 +0200
+++ cryptopp-5.6.5/GNUmakefile	2017-12-23 23:15:44.213704213 +0100
@@ -127,9 +127,9 @@
 
 # Guard use of -march=native
 ifeq ($(GCC42_OR_LATER)$(IS_NETBSD),10)
-   CXXFLAGS += -march=native
+   #CXXFLAGS += -march=native
 else ifneq ($(CLANG_COMPILER)$(INTEL_COMPILER),00)
-   CXXFLAGS += -march=native
+   #CXXFLAGS += -march=native
 else
   # GCC 3.3 and "unknown option -march="
   # Ubuntu GCC 4.1 compiler crash with -march=native
